src/sentry/issues/escalating_issues_alg.py

```diff
@@ -78,17 +78,6 @@ def generate_issue_forecast(
 
     ts_max = max(ts_data)
 
-    # minimum ceiling for issues that don't have much data from the past week
-    floor_bound = 200
```

Comment:
should this be set to 0 instead of removed?

Reply:
why set to 0?

Reply:
oh nevermind, ignore me